frontend/src/app/services/unified-config-wrapper/unified-config-wrapper.service.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, map, Observable, tap } from 'rxjs';
+import {
+  UnifiedConfig,
+  UnifiedConfigService as OpenAPIUnifiedConfigService,
+} from '../../openapi';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UnifiedConfigWrapperService {
+  constructor(private unifiedConfigService: OpenAPIUnifiedConfigService) {
+    this.loadUnifiedConfig().subscribe();
+  }
+
+  loadUnifiedConfig(): Observable<UnifiedConfig> {
+    return this.unifiedConfigService
+      .getUnifiedConfig()
+      .pipe(tap((unified) => this._unifiedConfig.next(unified)));
+  }
+
+  private _unifiedConfig = new BehaviorSubject<UnifiedConfig | undefined>(
+    undefined,
+  );
+
+  readonly unifiedConfig$ = this._unifiedConfig.asObservable();
+
+  readonly metadata$ = this.unifiedConfig$.pipe(
+    map((unifiedConfig) => unifiedConfig?.metadata),
+  );
+  get metadata() {
+    return this._unifiedConfig.value?.metadata;
+  }
+  readonly feedback$ = this.unifiedConfig$.pipe(
+    map((unifiedConfig) => unifiedConfig?.feedback),
+  );
+  get feedback() {
+    return this._unifiedConfig.value?.feedback;
+  }
+  readonly navbar$ = this.unifiedConfig$.pipe(
+    map((unifiedConfig) => unifiedConfig?.navbar),
+  );
+  get navbar() {
+    return this._unifiedConfig.value?.navbar;
+  }
+  readonly beta$ = this.unifiedConfig$.pipe(
+    map((unifiedConfig) => unifiedConfig?.beta),
+  );
+  get beta() {
+    return this._unifiedConfig.value?.beta;
+  }
```

Comment:
```suggestion
```

I'd prefer to keep the service simple in general and also remove all those different functions. If we add more and more, the service get's more messy. Instead, pipe the Observable in the components where they are actually used.